src/wallet/rpcdump.cpp

```diff
@@ -1737,3 +1737,69 @@ RPCHelpMan importdescriptors()
 },
     };
 }
+
+RPCHelpMan listdescriptors()
+{
+    return RPCHelpMan{
+        "listdescriptors",
+        "\nList descriptors imported into a wallet.",
```

Comment:
Should document whatever behaviour we end up with for non-descriptor wallets.

Reply:
For me it's kind of obvious and the error is clear. What do others think?

Reply:
Thanks. I've taken your suggestion and updated the description.